app/controllers/orchestration_stack_controller.rb

```diff
@@ -46,6 +46,10 @@ def resources
     show_association('resources', _('Resources'), :resources, OrchestrationStackResource)
   end
 
+  def children
```

Comment:
if you name this method `display_children`, none of the other changes should be needed

(`display_nested_list` checks for `display_@display` method, and falls back to the generic implementation only if it doesn't exist)

Reply:
Correct, fixed. Thanks!